remotespark/datawidgets/ipywidgetfactory.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2015  aggftw@gmail.com
+# Distributed under the terms of the Modified BSD License.
+
+from ipywidgets import VBox, Output, Button, HTML, HBox, Dropdown, Checkbox
+
+
+class IpyWidgetFactory(object):
```

Comment:
These class is doing actual work. The abstraction was done for unit testing/good design.